doc/articles/how-uno-works.md

```diff
@@ -2,60 +2,110 @@
+For every platform, the public API - panels, control templates, measurement and arranging, etc - is the same. However, it's sometimes useful to know what's going on at render-time, for example if you're [intermixing native views with XAML views](xref:Uno.Development.NativeViews) on a platform-specific basis.
 
-### iOS and Mac Catalyst
+### Skia Rendering
 
-On iOS and Mac Catalyst, all types that inherit from `Microsoft.UI.Xaml.FrameworkElement`, also inherit from the native [`UIView` type](https://learn.microsoft.com/dotnet/api/uikit.uiview). That is to say, on iOS, all XAML visual elements are also native views.
+Skia is a cross-platform, hardware-accelerated rendering engine used by Uno to draw the entire visual tree. Unlike native rendering, Skia doesn’t rely on platform UI components.
 
-When rendered at runtime, certain `FrameworkElement` types implicitly create inner views that inherit from higher-level native view types. For example, `Image` implicitly creates an inner `NativeImage` view, where `NativeImage` is an Uno-defined internal type that inherits directly from the native `UIKit.UIImageView` type.
+#### Platforms Supported
 
-### Android
+Skia rendering is available on:
 
-On Android, all types that inherit from `Microsoft.UI.Xaml.FrameworkElement`, also inherit from the native [`ViewGroup` type](https://learn.microsoft.com/dotnet/api/android.views.viewgroup). That is to say, on Android, all XAML visual elements are also native views.
+- **macOS**
+- **Linux**
+- **Windows** (called just Desktop, as an alternative to *native* WinUI rendering)
+- **WebAssembly**
+- **Android**
+- **iOS**
 
-When rendered at runtime, certain `FrameworkElement` types implicitly create inner views that inherit from higher-level native view types. For example, `Image` implicitly creates an inner `NativeImageView`, where `NativeImageView` is an Uno-defined internal type that inherits directly from the native `Android.Widget.ImageView` type.
+#### How Skia Rendering Works
 
-### Skia (Linux with X11/Wayland, Linux Framebuffer, macOS, Windows)
+- The entire UI is drawn onto a Skia canvas.
+- There are **no native views**; all visuals are composed in Skia using vector graphics.
+- A minimal native shell (like a window or web canvas) hosts the Skia surface.
 
-On Linux, XAML visual elements are rendered directly to a [Skia](https://skia.org/) canvas. Unlike the other target platforms, there's no 'native view type' to speak of. The Skia canvas is hosted inside of a bare minimum X11 shell.
+> [!NOTE]
+> Because it bypasses native UI components, Skia can offer pixel-perfect rendering and visual consistency, but it may lack native platform behaviors such as accessibility or platform styling.
```

Comment:
@morning4coffe-dev should link to the limitation section of the using-skia-rendering doc